src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,40 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  const selectedTab = tabs.find(({ id }) => id === selectedTabId) || tabs[0];
```

Comment:
You use the same search in two components, you should export it as a separate helper and reuse it in these components.